buildtools/u2204/env.inc

```diff
@@ -0,0 +1,44 @@
+export PATH="\
+/opt/circonus/perl/bin\
+:/opt/circonus/java/bin\
+:/opt/circonus/bin\
+:/usr/local/sbin\
+:/usr/local/bin\
+:/usr/sbin\
+:/usr/bin\
+:/sbin\
+:/bin"
+export CPPFLAGS="\
+ -I/opt/circonus/include\
+ -I/opt/circonus/include/luajit\
+ -I/opt/circonus/include/mysql\
+ -DGTEST_HAS_PTHREAD=1"
+export CFLAGS="\
+ -ggdb3\
+ -O2\
+ -Wall\
+ -Werror\
+ -Wno-deprecated-declarations\
+ -Wno-unused-variable\
+ -Wno-format-truncation\
+ -Wno-misleading-indentation\
+ -Wno-stringop-overflow"
+export CXXFLAGS="\
+ -ggdb3\
+ -O2\
+ -Wall\
```

Comment:
Please select a C++ standard

Reply:
not part of this PR (was divergent commits, have been resolved)